src/index.html

```diff
@@ -7,8 +7,87 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;500;600&family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a 
+        href="#home"
+        class="logo"
+      >
+        <img src="images/logo.png" alt="logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="nav_item">
+            <a 
+              href="#apple"
+              class="nav_link is-active"
+             >
+              Apple
+            </a>
+          </li>
+          
+          <li class="nav_item">
+            <a 
+              href="#samsung"
+              class="nav_link">
+              Samsung
+            </a>
+          </li>
+          
+          <li class="nav_item">
+            <a
+              href="#smartphones"
+              class="nav_link">
```

Comment:
```suggestion
            <a
              href="#smartphones"
              class="nav_link"
            >
```
pls, check it and fix everywhere